mne/epochs.py

```diff
@@ -467,12 +467,18 @@ def __init__(
     ):  # noqa: D102
         if events is not None:  # RtEpochs can have events=None
             events = _ensure_events(events)
-            events_max = events.max()
-            if events_max > INT32_MAX:
-                raise ValueError(
-                    f"events array values must not exceed {INT32_MAX}, "
-                    f"got {events_max}"
-                )
+            # Allow reading empty epochs (ToDo: Maybe not anymore in the future)
+            if events.shape == (0, 3):
```

Comment:
This is more explicit about what you're checking. You could also use `if events.size == 0` or `events.shape[0] == 0` if you want. (It makes it so you don't have to think "oh this could also evaluate to False if events.shape[1] is not equal to 3" and then realize that could never happen based on prior code. So cleaner just to check the thing you're actually interested in and ignore `shape[1]` entirely.)
```suggestion
            if len(events) == 0:
```